### PR TITLE
Adding Solo.io Cilium Workshop with Certification to Cilium.io Website.

### PR DESCRIPTION
**Pull request by distributethe6ix**
- [ ]  **Training partner must have a track record of providing high-quality trainings for open source technology and projects**

Solo.io has a number of workshops that are delivered on a weekly basis. 

All workshops are here: https://www.solo.io/events/upcoming/#workshops

These are the following workshops we deliver:
* Envoy Basics
* Istio, Getting Started
* Istio for Production
* Istio, Day 2 Operationalization
* Introduction to Cilium
* Gloo Mesh

- [ ]  **Trainings must cover Cilium Open Source or an [approved distribution](https://cilium.io/enterprise)**

Cilium is an open source software for providing, securing and observing network connectivity between container workloads - cloud native, and fueled by the revolutionary Kernel technology eBPF.

In this workshop, we will review the following:

- Deployment of Cilium (including hubble) on a KinD Kubernetes cluster
- Deployment of the bookinfo application
- Looking at the service to service communication using the Hubble UI and then looking at the gRPC endpoint which provides the data used by the UI to build the graph
- Leveraging network policies to secure service to service communications 

This workshop also includes a certification option. This “Fundamentals for Cilium” credential, offered by Solo.io with Credly, certifies that you possess the essential skills to deploy the Cilium CNI on a test Kubernetes cluster, gather metrics and enforce network policies. At the completion of the workshop, you will be able to take an assessment and a score 80% or higher earns the certification.

- [ ]  **The provided Cilium training will be reviewed by a Cilium committer to assess quality, accuracy, and completeness.**
Please let us know when you would like to review and we can send you an invite to take the workshop.

- [ ] **Have a landing page that includes information on your Cilium training.**
https://www.solo.io/solo-academy/workshops-cilium/

### CODE REVIEW

**Review by michi-covalent — APPROVED** [review]

**Comment by michi-covalent** [comment]
not sure if netlify is supposed to do something

**Comment by michi-covalent** [comment]
cc @xmulligan let's set up CODEOWNERS file so that we can get notified about pull requests on this repository.

**Comment by lizrice** [comment]
@distributethe6ix I've signed up for the workshop on 22nd Sep for the purpose of reviewing it as a committer

File: src/components/pages/enterprise/training/training.jsx
Comment (liquidat) [inline src/components/pages/enterprise/training/training.jsx:43]: The first sentence explains what Cilium is, without any relation to the workshop. There is no need to do that at this position, I'd drop the first sentence.

Comment (distributethe6ix) [reply src/components/pages/enterprise/training/training.jsx:43]: The first sentence explains what Cilium is, it's in every workshop that we deliver and I'm looking to maintain consistency. I am okay with leaving it.

Comment (lizrice) [reply src/components/pages/enterprise/training/training.jsx:43]: @distributethe6ix I think @liquidat's comment is aimed more about maintaining consistency with the rest of the Cilium site

**Comment by lizrice** [comment]
I had the opportunity to attend the workshop at OSS Summit EU. I had a few comments on the content, I think they should all be easy to address. (I also had a chat with @linsun and Krisztian Fekete who I think agreed these seem reasonable.) 

**Instruqt lab**
- In the workshop I attended there was an issue where some people saw Hubble pods getting stuck in ContainerCreating state after the Helm upgrade. Krisztian and Lin said that's something that started happening since recently upgrading the workshop to use Cilium 1.12.1. Let's get this sorted out - on the face of it, it sounds like it's a Cilium bug so if someone could raise an issue with some more info (or even a PR) that would be helpful.
- An audience member asked about L7 metrics and they got the answer that these are only available in Cilium Enterprise, which is incorrect - they are part of Cilium OSS. You do need to have something that enables [L7 visibility](https://docs.cilium.io/en/v1.12/policy/visibility/#proxy-visibility) (i.e. that starts the cilium-envoy process that you already show as part of this workshop). If you revisit the Grafana dashboard after the L7 policy is applied, and some traffic has flowed, you'll see some HTTP metrics generated there - that would be an easy change to the Instruqt lab that would avoid perpetuating the confusion about whether L7 metrics are included. 

**Slides**
- There are one or two diagrams in the presentation slides that come from ebpf.io, please could you add acknowledgement of the source (otherwise the copyright marker on the slides implies that Solo is the originator) 
- Where the presentation discusses eBPF programming, it covers BCC, libbpf / CO-RE, and Bumblebee. Given that this is a workshop about Cilium, please could you cover cilium/ebpf as well, since that's what Cilium uses.
- There was a Gloo Network slide at the end that I was too slow to really capture but I think it also implied that L7 metrics and also multi-cluster weren't part of Cilium OSS? 

Thanks!

File: src/components/pages/enterprise/training/training.jsx
Comment (lizrice) [inline src/components/pages/enterprise/training/training.jsx:41]:
```suggestion
    title: 'Introduction to Cilium',
```

We shouldn't imply any official certification (until we come up with something like the Cilium equivalent of the CKA of course!)

**Comment by xmulligan** [comment]
@distributethe6ix 👋  I just wanted to follow up on this. Have you had a chance to address the questions about the training or the comments in the PR?

If you want to discuss in next week at KubeCon, feel free to drop by the [Cilium Project Meeting](https://docs.google.com/document/d/1GXHNt3JrmvZY_uYI9sO4SHLej2E5SCpLpZsD0fpk3Ms/edit#)

**Comment by xmulligan** [comment]
Forgot to update this after our meeting on November 16th.

@distributethe6ix is going to do a few updates so we can get this merged
1) Update the title from Liz's suggestions above
2) Update the description to match other ones on the website
3) Give an update on the slides to show they have incorporated the feedback and now include attribution

**Comment by distributethe6ix** [comment]
> Forgot to update this after our meeting on November 16th.
> 
> @distributethe6ix is going to do a few updates so we can get this merged
> 
> 1. Update the title from Liz's suggestions above
> 2. Update the description to match other ones on the website
> 3. Give an update on the slides to show they have incorporated the feedback and now include attribution

@xmulligan @lizrice I have update the title, updated the description, update our solo.io logo to the latest one and also added an attribution to ebpf.io for the image we are using in our presentation deck. Please let me know if anything else is required to merge this request.

**Comment by christian-posta** [comment]
What's the status on this? is it ready to merge? Seems all of the previous asks have been satisfied.

File: src/components/pages/enterprise/training/training.jsx
Comment (xmulligan) [inline src/components/pages/enterprise/training/training.jsx:43]:
```suggestion
      'In this workshop, you will learn the essential skills to deploy the Cilium CNI on a test Kubernetes cluster, gather metrics and enforce network policies.,
```

Comment (xmulligan) [reply src/components/pages/enterprise/training/training.jsx:43]: to keep this in line with @lizrice 's comment above

File: src/components/pages/enterprise/training/training.jsx
Comment (xmulligan) [inline src/components/pages/enterprise/training/training.jsx:44]:
```suggestion
    buttonText: 'Start Workshop',
```

**Comment by xmulligan** [comment]
Was just going to follow up about this because I chatted with a few Solo folx about getting it merged at KubeCon. Looks like the deploy preview is currently failing. I've also added a few suggestions to bring it in line with the rest of the site.

**Comment by distributethe6ix** [comment]
> Was just going to follow up about this because I chatted with a few Solo folx about getting it merged at KubeCon. Looks like the deploy preview is currently failing. I've also added a few suggestions to bring it in line with the rest of the site.

Hi @xmulligan I've accepted your changes. Please go ahead and merge  (unless something else is pending, please let us know)

**Comment by xmulligan** [comment]
Thanks for the update! It looks like there are still some conflicts and the deploy preview is failing. Once those are green, I can merge it.

**Comment by distributethe6ix** [comment]
> 
@xmulligan can we hop on a call to discuss? I'm not sure where these changes are breaking. Was the structure of the website changed recently that might be contributing to these conflicts?

**Comment by xmulligan** [comment]
@distributethe6ix  Looks like one a quote was missing on the end of the line. I think the error you are running into is because your branch is old. Can you merge the main branch into current one? Or maybe just open a new PR?

I'm traveling the next two days, but could have a call on Monday?

**Comment by xmulligan** [comment]
@distributethe6ix I hope KubeHuddle went well! Are you still working on this?

**Comment by distributethe6ix** [comment]
> @distributethe6ix I hope KubeHuddle went well! Are you still working on this?

Hi Bill, I created a new PR per your recommendation. This should have all feedback captured from previously. 

https://github.com/cilium/cilium.io/pull/283

**Pull request closed without merging**
